src/features/textEditor/components/tipTapEditorConfigs.ts

```diff
@@ -44,8 +45,9 @@ export const EDITOR_EXTENSIONS: Extensions = [
   CollapsibleBlockNode,
   CollapsibleBlockContentNode,
   CollapsibleBlockSummaryNode,
-  ReferenceNode,
-  SentenceCompletionExtension,
+  citationNode,
```

Comment:
I think I started using camel case to differentiate the TipTap extensions from React components, but we should keep names consistent. I will switch back to pascal case since that's the convention that TipTap uses.